documentation/book/appendix_metrics.adoc

```diff
@@ -37,64 +37,154 @@ endif::Kubernetes[]
+It also discovers an Alertmanager instance in the same namespace.
+If you are using a different namespace, download the resource file and update it using the following command:
+
+[source,shell,subs="+quotes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+
+To define Prometheus jobs that will scrape the metrics data, you must apply the `ServiceMonitor` resource located in the provided `strimzi-service-monitor.yaml` file.
+Download this file using the following command:
 
 [source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus.yaml
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/strimzi-service-monitor.yaml
 
-The provided `alerting-rules.yaml` file creates a `ConfigMap` with sample alerting rules; download the resource file as follows:
+Currently, the Prometheus Operator only supports jobs that include an `endpoints` role for service discovery. To use another role, edit the `additionalScrapeConfigs` property in the `prometheus.yaml` configuration file. This takes the name of the `Secret` and the name of the property in a given `Secret` in which additional configuration is stored.
+To create this `Secret` resource, use the following command:
 
 [source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml > alerting-rules.yaml
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/additional-properties/prometheus-additional.yaml
+oc create secret generic additional-scrape-configs --from-file=prometheus-additional.yaml
+
+The provided `prometheus-rules.yaml` file creates a `PrometheusRule` with sample alerting rules. Download and update the resource file as follows:
 
-To deploy all these resources you can run the following.
+[source,shell,subs="+quotes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
+
+To deploy these resources, run the following commands:
 
 [source,shell,subs=attributes+]
 oc login -u system:admin
-oc apply -f alerting-rules.yaml
+oc apply -f strimzi-service-monitor.yaml
+oc apply -f prometheus-rules.yaml
 oc apply -f prometheus.yaml
 
 ifdef::Kubernetes[]
-==== Deploying on {KubernetesName}
+==== Deploying Prometheus on {KubernetesName}
```

Comment:
Again, do these differ in something more than just `oc` versus `kubectl`?

Reply:
Not at this place

Reply:
So if it does not differ, would it be easier to merge it into one chapter with two separate command sets?